docs/developer-guide/frontend.md

```diff
@@ -4,7 +4,7 @@
 
 To ensure we are working in a way that is compatible with the front end technology aspects of the [GOV.UK Service Manual](https://www.gov.uk/service-manual/technology).
 
-**All front end development should follow The National Archives [front end development guide](https://github.com/nationalarchives/front-end-development-guide)** on GitHub. This will ensure user interfaces are robust, inclusive and meet relevant regulations. 
+**All front end development should follow The National Archives [developer handbook](https://nationalarchives.github.io/developer-handbook/technology/)**. This will ensure user interfaces are robust, inclusive and meet relevant regulations. 
```

Comment:
The developer handbook is still a work in progress at the moment.

It's probably worth noting this as there is still a lot of good advice in the frontend development guide that hasn't yet been ported over but if they contradict one another, the developer handbook should be the one to follow.

Reply:
I've updated the text for this now so that both guides are linked to.